models/DataObject/ClassDefinition/Data/Localizedfields.php

```diff
@@ -1422,6 +1427,22 @@ public function getLabelWidth()
         return $this->labelWidth;
     }
 
+    /**
+     * @param string $labelAlign
+     */
+    public function setLabelAlign($labelAlign): void
```

Comment:
@blankse I'd prefer to move this code into a trait, so avoid code duplication, see also: 
https://github.com/pimcore/pimcore/blob/46b3bff3c519e8eb3254f8da016edb1ef9482755/models/DataObject/ClassDefinition/Layout/Traits/IconTrait.php#L19

Reply:
@brusch I did it: https://github.com/pimcore/pimcore/pull/6880/commits/dd5e420b23561aa79f5a31a5edf91a5fab4bf595

Reply:
Thank you 👍😊